motors_number: 36